lighthouse-core/audits/byte-efficiency/uses-request-compression.js

```diff
@@ -40,8 +41,12 @@ class ResponsesAreCompressed extends ByteEfficiencyAudit {
   static audit_(artifacts) {
     const uncompressedResponses = artifacts.ResponseCompression;
 
+    function skipUnrelatedResponses(record) {
+      return URL.getProtocol(record.url) !== CHROME_EXTENSION_PROTOCOL;
```

Comment:
Ah awesome :) you should have access to `protocol` if you end up moving it to the gatherer ;) https://github.com/GoogleChrome/lighthouse/pull/2733#discussion_r129152300